R/read_csv.R

```diff
@@ -0,0 +1,262 @@
+#' Check that the sampling information from two CSV files matches.
+#' Will throw errors if the sampling informations dont match. If
+#' it returns, the sampling informations match.
+#'
+#' @param a the first sampling info to check
+#' @param b the second sampling info to check
+#'
```

Comment:
Can you add `#' @noRd` so that this doc is kept internal? (`@export` controls whether the function is exported but the doc will still show up unless there's `@noRd`)